thr/settings/dev.py

```diff
@@ -15,7 +15,6 @@
 # pylint: disable=wildcard-import
 from .base import *
 DEBUG = True
-ALLOWED_HOSTS = ['localhost', '127.0.0.1']
```

Comment:
because it was duplicated as it's mentioned in `base.py` in [line 55](https://github.com/Ensembl/thr/blob/7bd6d427b0f517d5c452028073e802d70667cd2a/thr/settings/base.py#L55) and this one was causing issues when I was deploying to k8s

Reply:
Cool, makes sense